components/TextFields/src/MDCTextFieldPositioningDelegate.h

```diff
@@ -25,6 +25,15 @@
 
 @optional
 
+/**
```

Comment:
Tip: Consider documenting this as a convenience to calling the other method. May be helpful for subclasses to know.